### ISSUE TITLE
[IOS][release] Module 'audioplayers' not found

### ISSUE THREAD

**Issue by nguyenhuutinh** [luanpotter/audioplayers#308]
everything is ok when build on simulator

But when i archived it on xcode, this error occurs.

Do you know how to fix it?

XCode Version 11.2 (11B52)

**Comment by nguyenhuutinh** [comment]
<img width="1155" alt="Screenshot 2019-11-01 at 9 08 16 PM" src="https://user-images.githubusercontent.com/4114159/68026875-e15d1a00-fceb-11e9-9b2d-87008171c24d.png">

**Comment by nguyenhuutinh** [comment]
my fault https://github.com/flutter/flutter/issues/41033

**Comment by KapilPremware** [comment]
did you find any solution ?

**Comment by AmbujaAK** [comment]
I am getting the same issue, Any solution ?

**Comment by ologunB** [comment]
I have the same issue, any fix yet?

**Comment by ologunB** [comment]
> everything is ok when build on simulator
> 
> But when i archived it on xcode, this error occurs.
> 
> Do you know how to fix it?
> 
> XCode Version 11.2 (11B52)

Did you find a fix to it?

**Comment by peter100u** [comment]
> 在模拟器上构建时一切正常
> 
> 但是当我在 xcode 上归档它时，就会发生这个错误。
> 
> 你知道如何解决吗？
> 
> XCode 版本 11.2 (11B52)

我也面临同样的问题，有什么解决办法吗？请给出意见

**Comment by mannan541** [comment]
have the same issue. Any solution so far?

**Comment by peter100u** [comment]
是的,我已经解决了，但是我不知道为什么， 我升级了最新的Xcode版本，错误就消失了

**Comment by generalprognostics** [comment]
Incase some one is still looking for the solution, just make sure that that the target OS version in XCode and pod files are same.

**Comment by source-code-hub** [comment]
Just in case if anyone is still facing this problem, please make sure that the POD file target version and XCODE app target version are same.

**Comment by MilosKarakas** [comment]
This helped me:
1. Close Xcode
2. Open terminal
3. Navigate to your flutter project directory (my case: users/user/StudioProjects/<project-folder>)
4. Run "flutter clean" in terminal
5. Run "flutter pub get" in terminal
6. Run "cd ios" in terminal
7. Run "pod install"

After that I was able to build the app. It seems like pod would not get installed for some reason until I removed them all and started again.
